src/Integration.Vsix/CFamily/MessageHandler.cs

```diff
@@ -109,6 +112,8 @@ public void HandleMessage(Message message)
                     HandleAnalysisIssue(message);
                     break;
             }
+
+            CodeMarkers.Instance.HandleMessageStop();
```

Comment:
how does ETW know to correlate between the start and end? In my POC I passed a guid to both stop and start.

Reply:
No idea. It may not - it might be logic in PerfView that is doing the calculation. It's open source, so we should be able to find out.